packages/visual-editing/src/util/findSanityNodes.ts

```diff
@@ -176,3 +176,18 @@ export function findSanityNodes(
   }
   return elements
 }
+
+export function getSanityNodeArrayPath(sanityPath: string): string {
```

Comment:
this is quite a rough implementation currently. will test nested array items, etc, when working on the UI.